docs/api/browser-window.md

```diff
@@ -571,6 +571,10 @@ Enters or leaves the kiosk mode.
 
 Returns whether the window is in kiosk mode.
 
+### `win.getNativeWindowHandle()`
+
+Returns the OS window handle in a buffer.
```

Comment:
Can you document what exactly this returns on every OS, and how you would use it?